test/functional/admin/cabinet_ministers_controller_test.rb

```diff
@@ -47,4 +47,26 @@ def organisation
     assert_equal MinisterialRole.whip.order(:seniority).to_a, [role_2, role_1]
     assert_equal MinisterialRole.whip.order(:whip_ordering).to_a, [role_1, role_2]
   end
+
+  test 'should list ministerial organisations in ministerial order' do
+    org_1 = create(:ministerial_department, ministerial_ordering: 0)
+    org_2 = create(:ministerial_department, ministerial_ordering: 2)
+    org_3 = create(:ministerial_department, ministerial_ordering: 1)
+
+    get :show
+
+    assert_equal assigns(:organisations).to_a, [org_1, org_3, org_2]
```

Comment:
It's worth noting that Rails 5 [deprecates `assigns`](https://github.com/rails/rails/issues/18950) so ideally we should be moving away from this style of testing.